CoffeeChat/Controller/ProfileMenuViewController.swift

```diff
@@ -24,18 +25,27 @@ class ProfileMenuViewController: UIViewController {
         MenuOption(image: "settings", text: "Settings")
     ]
     private let profileImageSize = CGSize(width: 120, height: 120)
-
+    private var user: User!
```

Comment:
Can make this not a force unwrapped optional by moving `self.user = user` above the `super.init` call